example/sparse/wide_deep_model.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+from weighted_softmax_ce import *
+
+
+def wide_deep_model(num_linear_features, num_embed_features, num_cont_features, input_dims, hidden_units, positive_cls_weight):
+    data = mx.symbol.Variable("data", stype='csr')
+    label = mx.symbol.Variable("softmax_label")
+
+    x = mx.symbol.slice_axis(data=data, axis=1, begin=0, end=num_linear_features)
+    x = mx.symbol.cast_storage(x, 'csr')
```

Comment:
I think we should implement `slice_axis` for `csr` first. I don't think it's a good idea to add `cast_storage` in the network in the example.

Reply:
`cast_storage ` is removed now and this will cause `adam` fallback. I will work on `slice_axis ` for `csr` these days but it will not affect the codes here.